analyzers/src/SonarAnalyzer.CSharp/Rules/SymbolicExecution/SymbolicExecutionRunner.cs

```diff
@@ -108,6 +117,10 @@ protected override void Initialize(SonarAnalysisContext context)
         protected override void AnalyzeSonar(SyntaxNodeAnalysisContext context, bool isTestProject, bool isScannerRun, SyntaxNode body, ISymbol symbol)
         {
             var enabledAnalyzers = SonarRules.Where(x => x.SupportedDiagnostics.Any(descriptor => IsEnabled(context, isTestProject, isScannerRun, descriptor))).ToArray();
+            if (!Configuration.ForceSonarCfg)
+            {
```

Comment:
I can't comment on `IAnalyzerConfigurationExtensions`. The file was moved without changed - that doesn't seem to be needed in this PR